scripts/run_tests.sh

```diff
@@ -49,15 +37,7 @@ run_launch() {
   for dir in $(find ./dist-tests/$1-launch -type d)
   do
     cd $dir
-    torchrun \
-      --nproc_per_node 2 \
-      main.py --backend gloo --data_path ~/data \
-      --train_batch_size 2 \
-      --eval_batch_size 2 \
-      --num_workers 1 \
-      --max_epochs 2 \
-      --train_epoch_length 4 \
-      --eval_epoch_length 4
+    torchrun main.py ../../src/tests/ci-configs/$1-launch.yaml
```

Comment:
This command is not the same as the old one:
```suggestion
    torchrun --nproc_per_node 2 main.py ../../src/tests/ci-configs/$1-launch.yaml --backend gloo
```